<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="no_cd_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>无冷却</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cd_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭无冷却</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mod_sunlight_input">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="mod_sunlight_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修改阳光</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>阳光</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>冷却</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>金币</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mod_coin_input">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="mod_coin_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修改金币</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mod_sunlight_auto_checkbox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>自动修改 (3s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="handle_find_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>寻找游戏进程</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>类名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>窗口名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="class_name_input">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="window_name_input">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>植物大战僵尸杂交版v2.0.88</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAuthor">
    <property name="title">
     <string>作者</string>
    </property>
    <addaction name="actionZJamss"/>
    <addaction name="actionhttps_github_com_ZJamss"/>
   </widget>
   <addaction name="menuAuthor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionZJamss">
   <property name="text">
    <string>ZJamss</string>
   </property>
  </action>
  <action name="actionhttps_github_com_ZJamss">
   <property name="text">
    <string>https://github.com/ZJamss</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
